### PR TITLE
Added setter for tab text colors

### LINKED ISSUE #10: Cannot programatically set tab text colors
There is no publicly accessible API to set the tab text colors as there is for indicators and dividers.

### PR DESCRIPTION
**Pull request by Numan1617**
Fixes #10

### CODE REVIEW

**Comment by ogaclejapan** [comment]
Hi, @Numan1617 
Thank you for PR

Please pull request of only 1c4228c and 3789af2.
.gitignore file I will fix later.

**Comment by Numan1617** [comment]
Hi @ogaclejapan,

I've just forced pushed to update the PR without the gitignore changes.

File: library/src/main/java/com/ogaclejapan/smarttablayout/SmartTabLayout.java [hunk 1]
```diff
@@ -206,6 +206,17 @@ public void setCustomTabColorizer(TabColorizer tabColorizer) {
...
     }
 
+
+    /**
+     * Sets the colors used for styling the tab text. This will need to be called prior to calling
+     * {@link #setViewPager(android.support.v4.view.ViewPager)} otherwise it will not get set
+     *
+     * @param tabTextColors ColorStateList to use for tab text
+     */
+    public void setTabTextColors(ColorStateList tabTextColors) {
```
Comment (ogaclejapan) [inline library/src/main/java/com/ogaclejapan/smarttablayout/SmartTabLayout.java]: I'm sorry, could you change the method name?

```java
public void setDefaultTabTextColor(ColorStateList colors) {
    //....
}
```

I want to define a standard of TextView in the same way :D
http://developer.android.com/reference/android/widget/TextView.html#setTextColor(android.content.res.ColorStateList)

**Comment by ogaclejapan** [comment]
@Numan1617 Thank you :+1:

**Comment by Numan1617** [comment]
No worries. Whats the usual release cycle to maven for changes?

**Comment by ogaclejapan** [comment]
Hi, @Numan1617
For now, planning to release the next version within one day.

**Pull request merged**